### ISSUE TITLE
Accompanying alt text for multiple file uploads

### ISSUE THREAD

**Issue by dvodvo** [carrierwaveuploader/carrierwave#2639]
While carrierwave allows the uploading of mutiple files using a JSON data type (postgresql and mysql), AFAIK the rails file_field does not have an accessor in order to set the `alt`  attribute for the tag in order to support accessibility.

With a json column, it would make sense to create a has with the image data and atl_text as distinct elements.

Is there any way to achieve this?

**Comment by mshibuya** [comment]
There's no ready-made way to do so. Still you can store alt text (or whatever you want) in your model attributes, so it can be used later.
